embrace-android-sdk/api/embrace-android-sdk.api

```diff
@@ -1,3 +1,7 @@
+public final class io/embrace/android/embracesdk/AutoStartInstrumentationHook {
```

Comment:
Ideally this would be under an `internal` package (but have public visibility) and the function would not contain an underscore